functions/core.sh

```diff
@@ -37,9 +37,11 @@ function flash_adb() {
 }
 
 function flash_rsync() {
+	TARGET_ARCH=$(adb shell uname -m)
```

Comment:
Not a fix for that yet: On ilyabizyaev's device, `uname -m` reports `i686`

Reply:
Theoretically, you could just replace all i*86 architectures with i386 (or x86)

Reply:
I wanted to have a solution without mapping different architecture shemes, but as their doesn't seem to be one, I'll probably do it like that.